.travis.yml

```diff
@@ -1,4 +1,4 @@
-bundler_args: --without benchmark:site:development
+bundler_args: --without benchmark:development
```

Comment:
Can we change this in the matrix, so `site` continues to be excluded for all other jobs?

Reply:
I don't think that's possible..
Besides, why is that a problem, Travis is going to use the cached gemset anyways..?